css/partials/_menu.scss

```diff
@@ -15,8 +15,9 @@
     width: 100%;
     a {
       width: 100%;
-      &.hover {
-	      color: #fff;
+      &.hover,
```

Comment:
Actually, if you notice, that period should be a colon, eh? And maybe that is why the hover on subpages wasn't working?